src/v0/util/index.test.js

```diff
@@ -456,3 +456,44 @@ describe('Unit test cases for combineBatchRequestsWithSameJobIds', () => {
     expect(combineBatchRequestsWithSameJobIds(input)).toEqual(expectedOutput);
   });
 });
+
+describe('validateEventAndLowerCaseConversion Tests', () => {
```

Comment:
add test case for boolean values, what should happen in that cases for true and false

Reply:
it will throw error as our utility always converts to string